bin/fin

```diff
@@ -4525,10 +4593,11 @@ install_tools_debian ()
 
 	# Install Docker Compose
 	if ! is_docker_compose_version; then
-		echo-green "Installing Docker Compose..."
+		local dc_pkg=$(basename "$URL_DOCKER_COMPOSE_NIX")
+		echo-green "Installing docker-compose v${REQUIREMENTS_DOCKER_COMPOSE}..."
```

Comment:
Not a critical thing, but I guess we should always call it "downloading $toolname" rather than "installing $toolname" to prevent freaking people out

Reply:
Updated wording